src/view/dataElements/xdmObject/components/objectEdit.jsx

```diff
@@ -57,6 +57,11 @@ const ObjectEdit = ({ fieldName }) => {
             width="size-5000"
           />
         </DataElementSelector>
+      ) : (
+        <div>
+          Please use the tree to the left to provide values for individual
+          attributes.
```

Comment:
What does "use the tree" mean?

Please click on an attribute in the tree on the left to provide values for individual attributes.